modules/ROOT/pages/cloudhub-connector.adoc

```diff
@@ -41,34 +37,34 @@ image::cloudhub-connector-props-blank.png[]
 . For this demo, select *Basic Authentication* configuration, which must be configured with the corresponding *Username* and *Password*.
 +
 [IMPORTANT]
-*Inherited Token Authentication* is a new configuration option that works only when your CloudHub connector's application is run inside a CloudHub instance. With this approach the connector retrieves the API Token based on the user that created first the application. Note that if the user that created the application is disabled this authentication method will fail.
+*Inherited Token Authentication* is a configuration option that works only when your CloudHub connector's application is run inside a CloudHub instance. With this approach the connector retrieves the API Token based on the user that created first the application. If the user who created the application is deleted, this authentication method fails.
 +
-. Enter your credentials into the configuration fields and click *OK*:
+. Enter your Anypoint Platform credentials into the configuration fields and click *OK*. The *Url* field specifies the path to CloudHub in an Anypoint Platform control plane. For example, you can change the *Url* field to access other control planes such as EU. The EU control plane is at `+https://eu1.anypoint.mulesoft.com/+`.
 +
-image::cloudhub-connector-a71a6.png[cloudhub-basic-auth-config]
+image::cloudhub-connector-a71a6.png[]
 +
 . Make sure that you have selected the *Create notification* operation and set the properties for:
-.. *Domain*: This should be the domain you set for your application when you first created it following the xref:general:getting-started:deploy-to-cloudhub.adoc[Deploy to CloudHub tutorial]. Note that this is not the complete domain name (myapp.cloudhub.io) but only the application name (myapp)
-.. *Message*: The message you want your notification to show (e.g. "_Your application has been hit_"). In the example below, the notification will simply display a "_Hello World!_" message.
-.. *Priority*: The type of the notification. By selecting "INFO", your notification will display an informative message while selecting "ERROR" will display a red alert message.
-
+.. *Domain*: This should be the domain you set for your application when you first created it following the xref:general:getting-started:deploy-to-cloudhub.adoc[Deploy to CloudHub tutorial]. Note that this is not the complete domain name (myapp.cloudhub.io) but only the application name (myapp).
+.. *Message*: The message you want your notification to show (for example, "Your application has been hit"). In the example below, the notification will simply display a "Hello World!" message.
+.. *Priority*: The type of the notification. By selecting "INFO", your notification displays an informative message while selecting "ERROR" displays a red alert message.
```

Comment:
grammar: The notification is not selecting INFO. Might change this to something like 'When you select "INFO," your notification displays...'  I'm not sure we need to put quotes around INFO and ERROR.